server/app/services/twilio_sms_service.py

```
@@ -0,0 +1,277 @@
+class TwilioSMSService:
+    """Service for sending SMS messages via Twilio API."""
+    
+    def __init__(self):
+        """Initialize Twilio client with credentials from settings."""
+        try:
+            self.account_sid = settings.TWILIO_ACCOUNT_SID
+            self.auth_token = settings.TWILIO_AUTH_TOKEN
+            self.from_number = settings.TWILIO_PHONE_NUMBER
+            
+            if not all([self.account_sid, self.auth_token, self.from_number]):
+                raise ValueError("Missing required Twilio configuration")
```

Ruff (0.14.8)

Abstract `raise` to an inner function

(TRY301)